task_1/index.html

```diff
@@ -1,47 +1,140 @@
 <!DOCTYPE html>
 <html lang="en">
-
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link rel="stylesheet" href="style.css">
 </head>
-
-<body>   
+<body> 
     <div class="wrapper">
-        <div class="header"></div>
+        <div class="header">
+            <div class="coffee-logo">
+                <div><img class="coffee-image-logo" src="./photo/coffee logo.png" alt="coffee logo"></div>
+            </div>
+            <div class="navbar">
+                <div id="home-item" class="header-item active" onclick="onMenuItemClick('home-item')">Home</div>
+                <div id="about-item" class="header-item" onclick="onMenuItemClick('about-item')">About Us</div>
+                <div id="lands-item" class="header-item" onclick="onMenuItemClick('lands-item')">Our Lands</div>
+                <div id="suppliers-item" class="header-item" onclick="onMenuItemClick('suppliers-item')">Our Suppliers</div>
+                <div id="products-item" class="header-item" onclick="onMenuItemClick('products-item')">Products</div>
+                <div id="contact-item" class="header-item" onclick="onMenuItemClick('contact-item')">Contact Us</div>
+            </div>
+            <div class="header-images">
+                <div><img class="shopping-image" src="./photo/shopping cart.png" alt="shopping cart"></div>
```

Comment:
use kabab case for image name, ids and class

apply it everywhere 
```suggestion
                <div><img class="shopping-image" src="./photo/shopping-cart.png" alt="shopping cart"></div>
```